app/controllers/pages_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'iex-ruby-client'
```

Comment:
If you already added the gem for `iex-ruby-client` then this is not necessary to put this here